src/views/Dashboard.vue

```diff
@@ -114,50 +116,54 @@
               </v-card>
             </section>
 
-            <!-- Current Directors-->
-            <section v-if="!isSoleProp && !isPartnership">
+            <!-- Proprietor / Partners -->
+            <section v-if="isFirm">
```

Comment:
I moved the "is True" condition first and the "else" condition below. I think it's easier to read/understand this way: First, what does this apply to? Then, what's the fallback?